src/main/java/uk/ac/ox/oxfish/model/scenario/TunaScenario.java

```diff
@@ -746,6 +747,12 @@ public void setFadMortalityIncludedInExogenousCatches(boolean fadMortalityInclud
                     EVERY_YEAR,
                     IterativeAveragingAccumulator::new,
                     fadSet -> fadSet.getStep() - fadSet.getTargetFad().getStepDeployed()
+                ),
+                new BasicMonitor<>(
```

Comment:
is this basically a counter + sensor + gatherer?

Reply:
That's a good description. I needed some way to monitor various things about the purse seine actions and to group them by species and/or regions.